crates/crates_io_tarball/src/manifest.rs

```diff
@@ -84,3 +84,17 @@ impl IsInherited for DepsSet {
         self.iter().any(|(_key, dep)| dep.is_inherited())
     }
 }
+
+pub trait EditionExt {
+    fn as_str(&self) -> &'static str;
+}
+
+impl EditionExt for cargo_manifest::Edition {
+    fn as_str(&self) -> &'static str {
+        match self {
+            Self::E2015 => "2015",
+            Self::E2018 => "2018",
+            Self::E2021 => "2021",
+        }
+    }
+}
```

Comment:
this is essentially a polyfill for https://github.com/LukeMathWalker/cargo-manifest/pull/58 :)